app/models/order.rb

```diff
@@ -3,4 +3,5 @@ class Order < ApplicationRecord
   belongs_to :reward
 
   enum delivery_status: { ordered: 'ordered', delivered: 'delivered' }
+  scope :filter_by_delivery_status, ->(delivery_status) { where delivery_status: delivery_status }
```

Comment:
I think it is enough to scope this by `enum`, you can call `Order.ordered` or `Order.delivered` and it should return collection with items only having this enum.

[Look at scopes](https://api.rubyonrails.org/v5.1/classes/ActiveRecord/Enum.html)

Reply:
True, I don't need this scope - removed :)